### PR TITLE
WT-12930 Fix Coverity bugs in the model

### PR DESCRIPTION
**Pull request by pmacko86**
The PR addresses several Coverity errors and adds a missing include to make VS Code happy.

### CODE REVIEW

File: test/model/test/common/include/model/test/wiredtiger_util.h [hunk 1]
```diff
@@ -209,36 +213,43 @@ wt_rollback_to_stable(WT_CONNECTION *conn)
...
+        auto ret_b = (b);                                                         \
         if (ret_a != ret_b) {                                                     \
             std::cerr << std::endl;                                               \
             std::cerr << "Assertion failure in " << __PRETTY_FUNCTION__           \
                       << ": Values are not equal. " << std::endl;                 \
             std::cerr << "  Location: " << __FILE__ ":" << __LINE__ << std::endl; \
-            std::cerr << "  " << #a << ": " << ret_a << std::endl;                \
-            std::cerr << "  " << #b << ": " << ret_b << std::endl;                \
+            std::cerr << "  " << (label_a) << ": " << ret_a << std::endl;         \
+            std::cerr << "  " << (label_b) << ": " << ret_b << std::endl;         \
```
Comment (korteland) [inline test/model/test/common/include/model/test/wiredtiger_util.h:229]: What did Coverity dislike about this? I think the code was easier to read and simpler before.

Comment (pmacko86) [reply test/model/test/common/include/model/test/wiredtiger_util.h:229]: I agree that it was easier to read before. Coverity didn't like that the code didn't use `std::move` and created two issues for each usage of `wt_model_assert`, which was a lot of issues! I didn't want `std::move` to show up in the output to `cerr`, so I split the macro accordingly.

Comment (korteland) [reply test/model/test/common/include/model/test/wiredtiger_util.h:229]: Ouch. Fair enough. Thanks for the explanation.

**Review by korteland — APPROVED** [review]
LGTM!

**Pull request merged**